src/main/java/com/google/codeu/servlets/UserListServlet.java

```diff
@@ -0,0 +1,22 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Handles fetching all users for the community page.
+ */
+@WebServlet("/user-list")
+public class UserListServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
+
+    response.getOutputStream().println("this will be my user list");
+  }
+}
```

Comment:
Please add a newline so the GitHub newline thingy goes away :)

Reply:
resolved